### DISCUSSION TITLE
Problems with ARS

### DISCUSSION THREAD

**Discussion by Julu2106** [eccentricdevotion/TARDIS#676] [Q&A]
Hello everyone!
I've been using the plugin for some time now and never had any problems with it. But now I can't grow any new rooms, the plugin keeps telling me "You cannot grow rooms unless your TARDIS was created in it's own world!" That never came up before yesterday. In fact I even grew a new room the day before yesterday. Please send help!

**Comment by eccentricdevotion** [comment]
Most probably the TimeVortex world is being loaded by another world loader such as Multiverse. If so remove it from the MV config.

**Reply by eccentricdevotion** [reply]
See #653
